### ISSUE TITLE
Dave doesn't suck

### ISSUE THREAD

**Issue by DaveN53** [yupferris/fart-os#1]
# FuckFerris

**Comment by yupferris** [comment]
This is true now, though when it was written it wasn't necessarily the case. Hard to say what the original dev's were thinking at that time.

I think for the sake of backwards-compatibility we should continue to support the "legacy" code as-is.

**Comment by DaveN53** [comment]
Best guess is too much beer and starcraft. I suggest using the principles of PRVR in future releases to resolve this issue.

**Comment by yupferris** [comment]
Indeed. As long as it's not ... haha. ... _rushed_ ... hahahaha
